src/components/ContentHighlights/ContentHighlightsDashboard.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import ZeroStateHighlights from './ZeroStateHighlights';
+import CurrentContentHighlights from './CurrentContentHighlights';
+import { TEST_COURSE_HIGHLIHTS_DATA } from './data/constants';
+
+const ContentHighlightsDashboard = () => {
+  const hasContentHighlights = TEST_COURSE_HIGHLIHTS_DATA.length > 1;
+  // const hasContentHighlights = false;
```

Comment:
See other comments about potentially calling a custom React hook here.

Reply:
Leaving open for future reference